src/core/webview/ClineProvider.ts

```diff
@@ -1262,6 +1262,7 @@ export class ClineProvider extends EventEmitter<ClineProviderEvents> implements
 			customModePrompts,
 			customSupportPrompts,
 			enhancementApiConfigId,
+			autocompleteApiConfigId,
```

Comment:
We won't need this for the hyper MVP, but it's the start of the config we'd need. We can remove it for this PR, but keep it around for the future